### PR TITLE
Remove Maven in Gradle script

### LINKED ISSUE #410: Remove unnecessary gradle configuration that impacts gradle compatibility
**Is your feature request related to a problem? Please describe.**
The Anrdoid build.gradle file seems to have some unnecessary configuration that reduces gradle compatibility.

**Describe the solution you'd like**
No build errors about unused parts of the gradle configuration in Babylon React Native from an app consuming it and using a different/newer version of gradle than the Babylon React Native Playground app.

**Describe alternatives you've considered**
We should go through the gradle file, especially this section: https://github.com/BabylonJS/BabylonReactNative/blob/master/Package/Android/build.gradle#L123-L166, and remove whatever parts are unnecessary. 

**Additional context**
Most recently, we discovered that the `installArchives` block seems to be unneeded yet breaks compatibility with gradle 7+.

### PR DESCRIPTION
**Pull request by CedricGuillemet**
fixes #410 

related https://github.com/BabylonJS/BabylonReactNative/issues/388

Tested with 0.65 on MacOS

### CODE REVIEW

**Comment by ryantrem** [comment]
Don’t we need similar changes in https://github.com/BabylonJS/BabylonReactNative/blob/master/Package/Android/build.gradle?

**Review by pingumen96 — APPROVED** [review]
I tried this and everything works.

**Review by 3GOMESz — APPROVED** [review]

**Comment by CedricGuillemet** [comment]
> https://github.com/BabylonJS/BabylonReactNative/blob/master/Package/Android/build.gradle

Correct me if I'm wrong. The gradle you mentioned is used to build the package. If, for some reason, we had to build it with RN 0.68 (and 0.67), we would have to do the change. But 0.65 is compatible with 0.67 and 0.68, so it's not needed. I think it might be needed for 0.69 as it not ABI compatible.

**Comment by ryantrem** [comment]
> > https://github.com/BabylonJS/BabylonReactNative/blob/master/Package/Android/build.gradle
> 
> Correct me if I'm wrong. The gradle you mentioned is used to build the package. If, for some reason, we had to build it with RN 0.68 (and 0.67), we would have to do the change. But 0.65 is compatible with 0.67 and 0.68, so it's not needed. I think it might be needed for 0.69 as it not ABI compatible.

The build.gradle file I referenced is the one that is included in the npm package for the Android build. See https://github.com/BabylonJS/BabylonReactNative/blob/master/Package/gulpfile.js#L278

**Comment by CedricGuillemet** [comment]
> > > https://github.com/BabylonJS/BabylonReactNative/blob/master/Package/Android/build.gradle
> > 
> > 
> > Correct me if I'm wrong. The gradle you mentioned is used to build the package. If, for some reason, we had to build it with RN 0.68 (and 0.67), we would have to do the change. But 0.65 is compatible with 0.67 and 0.68, so it's not needed. I think it might be needed for 0.69 as it not ABI compatible.
> 
> The build.gradle file I referenced is the one that is included in the npm package for the Android build. See https://github.com/BabylonJS/BabylonReactNative/blob/master/Package/gulpfile.js#L278

My bad, you are right. Doing the change now.

File: Apps/Playground/0.64/ios/Playground.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -403,7 +403,7 @@
 			);
 			inputPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-Playground/Pods-Playground-frameworks.sh",
-				"${PODS_XCFRAMEWORKS_BUILD_DIR}/OpenSSL/OpenSSL.framework/OpenSSL",
+				"${PODS_XCFRAMEWORKS_BUILD_DIR}/OpenSSL-Universal/OpenSSL.framework/OpenSSL",
```
Comment (ryantrem) [inline Apps/Playground/0.64/ios/Playground.xcodeproj/project.pbxproj:406]: Does this change relate to removing maven?

File: Apps/Playground/0.64/ios/Playground.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -879,7 +879,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```
Comment (ryantrem) [inline Apps/Playground/0.64/ios/Playground.xcodeproj/project.pbxproj:882]: Same question here.

File: Apps/Playground/0.64/ios/Podfile.lock [hunk 3]
```diff
@@ -256,7 +256,7 @@ PODS:
   - React-jsinspector (0.64.1)
   - react-native-babylon (0.0.1):
     - React
-  - react-native-slider (4.0.0-rc.3):
+  - react-native-slider (4.2.3):
```
Comment (ryantrem) [inline Apps/Playground/0.64/ios/Podfile.lock:259]: And here :)

**Comment by CedricGuillemet** [comment]
@ryantrem Sergio did some changes because builds are failing for a few weeks now. @SergioRZMasson can you please elaborate on your changes and why it was failing actually ?

**Comment by SergioRZMasson** [comment]
@ryantrem , we has some conflicts with the npm packages been used by 0.64. I had to update some of them to get npm install working again. The other changes might be due to the fact that I was using a arm64 mac, I think the changes made to project.pbxproj can be ignored, but the Podfile.lock is required since we have changed some dependencies versions.

**Comment by ryantrem** [comment]
> @ryantrem , we has some conflicts with the npm packages been used by 0.64. I had to update some of them to get npm install working again. The other changes might be due to the fact that I was using a arm64 mac, I think the changes made to project.pbxproj can be ignored, but the Podfile.lock is required since we have changed some dependencies versions.

Ok, thanks for that context. Any specific reason those fixes are going into this PR, as opposed to being in a separate PR specifically about fixing the npm package ref issues?

**Review by ryantrem — APPROVED** [review]

**Review by 3GOMESz — APPROVED** [review]

**Comment by SergioRZMasson** [comment]
@ryantrem  It was mainly because it was failing the CI

**Pull request merged**